tpot/base.py

```diff
@@ -79,12 +80,15 @@ def handler(dwCtrlType, hook_sigint=_thread.interrupt_main):
 class TPOTBase(BaseEstimator):
     """Automatically creates and optimizes machine learning pipelines using GP."""
 
+    # dont save periodic pipelines more often than this
+    OUTPUT_BEST_PIPELINE_PERIOD_SECONDS = 30
```

Comment:
it's initial target is not to be user configurable - but to prevent too many calls to trying to export. in any case the limit is per generation.

if we one day allow the user to save pipeline after each single evaluation for example, then we might want this (or even might NOT want this).

In any case IMHO it seems a bit early to let this be a parameter.